robottelo.properties.sample

```diff
@@ -58,13 +58,24 @@ ssh_key=
 # Binary location for selected wedriver
 # webdriver_binary=/usr/bin/firefox
 # webdriver_binary=/usr/bin/chromedriver
-# cdn=true
-# Run one datapoint or multiple datapoints for tests
-# run_one_datapoint=false
+
+# webdriver_desired_capabilities accepts extra configuration to be passed
+# to saucelabs browser in order to configure the platform. You can use
+# the platform configurator (see link below) to get the information about
+# the environment you want to test.
+# https://wiki.saucelabs.com/display/DOCS/Platform+Configurator/
+# PS.: the base DesiredCapabilities dict will be get by the browser
+# specified by webdriver config. If you override browserName then that
+# browser will be used instead.
+#webdriver_desired_capabilities=browserName=firefox,platform=OS X 10.11,version=44.0,screenResolution=1024x768
```

Comment:
1) Can we specify what parameters are required for proper run? Do we need only these four
2) Do we have a list of possible parameters, options? Can we provide URL here as `https://wiki.saucelabs.com/display/DOCS/Platform+Configurator/` is not informative at all.